backend/shared/src/onboarding-helpers.ts

```diff
@@ -105,17 +108,22 @@ const sendNextDayManaBonus = async (
       description: 'Signup bonus',
       data: {},
     }
-    const manaBonusTxn = await runTxnFromBank(transaction, signupBonusTxn)
-    if (manaBonusTxn.status != 'error' && manaBonusTxn.txn) {
-      transaction.update(toDoc, {
-        manaBonusSent: true,
-        weeklyTrendingEmailSent: true, // not yet, but about to!
+
+    const manaBonusTxn = await pg
+      .tx((tx) => {
+        return runTxnFromBank(tx, signupBonusTxn)
```

Comment:
Seems like this nests the firestore transactions. That would break?

Reply:
Ok, this does a firestore write not a txn.

Reply:
[c] Should test creating a user to make sure this works [J] Yes

Reply:
it did break but now it's all un-nested